eng/jacoco-test-coverage/pom.xml

```diff
@@ -48,7 +48,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-ai-formrecognizer</artifactId>
-      <version>4.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-ai-formrecognizer;current} -->
+      <version>4.0.2</version> <!-- {x-version-update;com.azure:azure-ai-formrecognizer;current} -->
```

Comment:
We don't need to update the versions for Form Recognizer as the [increment versions PR](https://github.com/Azure/azure-sdk-for-java/pull/31439) for this library was already merged into `main`.
```suggestion
      <version>4.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-ai-formrecognizer;current} -->
```